client/__test__/actions/createGroup.test.jsx

```diff
@@ -0,0 +1,8 @@
+import expect from 'expect';
+import createGroup from '../../actions/createGroup';
```

Comment:
The filename for this file doesn't seem to tally with that of the other actions. They seem to have a suffix `Action` which this doesn't have.